presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -3336,6 +3348,16 @@ public static boolean isRewriteExpressionWithConstantEnabled(Session session)
         return session.getSystemProperty(REWRITE_EXPRESSION_WITH_CONSTANT_EXPRESSION, Boolean.class);
     }
 
+    public static boolean isEagerPlanValidationEnabled(Session session)
+    {
+        return session.getSystemProperty(EAGER_PLAN_VALIDATION_ENABLED, Boolean.class);
+    }
+
+    public static int getEagerPlanValidationThreadPoolSize(Session session)
```

Comment:
We can keep this as a config property, since it should be static throughout the cluster's uptime.

Reply:
Ok, so I kept the config in  FeaturesConfig, let me know if there is a better place for it